requirements.txt

```diff
@@ -49,6 +50,6 @@ traitlets==4.3.2
 typed-ast==1.1.0
 urllib3==1.23
 wcwidth==0.1.7
-Werkzeug==0.14.1
+Werkzeug==0.15.6
 wrapt==1.10.11
-xlrd>= 1.0.0
\ No newline at end of file
+xlrd>= 1.0.0
```

Comment:
Not sure what happened here; whitespace strip?

Reply:
Your editor probably added a trailing newline, which @Marc-Andre-Rivet's files often don't have. No worries, I do that all the time too 😏